WC_Pagantis.php

```diff
@@ -31,7 +29,8 @@
 define('PG_CONFIG_TABLE_NAME', 'pagantis_config');
 define('PG_LOGS_TABLE_NAME', 'pagantis_logs');
 define('PG_CONCURRENCY_TABLE_NAME', 'pagantis_concurrency');
-define('PG_CART_PROCESS_TABLE', 'cart_process');
+define('PG_OLD_CART_PROCESS_TABLE', 'cart_process');
+define('PG_ORDER_PROCESS_TABLE', 'pagantis_order');
```

Comment:
If you want to create a new table with "better" name, please rename the table initially, then you will to maintain only one table. Use this mysql command: https://www.techonthenet.com/mysql/tables/alter_table.php#:~:text=The%20syntax%20to%20rename%20a%20table%20in%20MySQL%20is%3A,The%20table%20to%20rename.